src/styles/blocks/card.scss

```
@@ -74,7 +74,8 @@
     align-items: center;
     color: $hover-color;
     text-transform: uppercase;
-    border: 1px solid #ccc transparent;
+    border: 1px solid #ccc;
+    border-color: transparent;
```

```suggestion
    border: 1px solid #ccc;
    border-color: transparent;
```

First, you set the color to `#ccc`, next you overwrite it with `transparent`. Why do you need to do it? Set only the color you need.